src/main/scala/nl/biopet/tools/bamstats/generate/Args.scala

```diff
@@ -25,7 +25,11 @@ import java.io.File
 
 case class Args(outputDir: File = null,
                 bamFile: File = null,
+                sample: String = null,
+                library: String = null,
+                readgroup: String = null,
                 referenceFasta: Option[File] = None,
-                binSize: Int = 10000,
-                threadBinSize: Int = 1000000,
+                bedFile: Option[File] = None,
+                excludePartialReads: Boolean = false,
+                getUnmappedReads: Boolean = false,
```

Comment:
Should this not true by default?

Reply:
If you are interested in a particular region, you do not want all unmapped reads.

If you want the whole bamfile and no regions are specified, unmapped reads are included.

Reply:
I will try to make the CLI more clear